README.md

```diff
@@ -21,9 +21,6 @@ $ export GITHUB_TOKEN=<paste in GitHub access token>
 $ ./services promote --from https://github.com/organisation/first-environment.git --to https://github.com/organisation/second-environment.git --service service-a --commit-name <User to commit as> --commit-email <Email to commit as>
 ```
 
-If the `commit-name` and `commit-email` are not provided, it will attempt to find them in `~/.gitconfig`, otherwise it will fail.
```

Comment:
I figured it was superfluous - we go about saying --from, --to, that kinda thing, but we don't go ahead and say `if these are not provided, it will fail` - and because .gitconfig isn't a thing I didn't see a point in mentioning that

Reply:
Not quite sure what `.gitconfig isn't a thing` means?

Reply:
Basically, that message means that if you don't provide the options, it will use the local configuration on the machine, which means that if you're running it locally _on your machine_ it will pick up the config from _your machine_.

Reply:
> Not quite sure what `.gitconfig isn't a thing` means?

Ah - I was referring to this in the context of the examples I've updated where I've removed the artifact (named gitconfig).

I'll go back and reword this.